src/reducers/index.ts

```diff
@@ -18,9 +42,64 @@ function reducer(state: Readonly<StateBase>, action: Action): StateBase {
   };
 }
 
+/**
+ * Exclude these actions from the history completely.
+ */
+const excludeActionsInHistory = [DATASET_INLINE_RECEIVE,
```

Comment:
A few questions

1) I understand that RESULT_RECEIVE/REQUEST is automatically triggered by spec change in the DatasetSelector and EncodingPane components, so they will get reapplied when a relevant redo is triggered.  

However, I'm not sure how `DATASET_INLINE_RECEIVE` get re-triggered.
Do we filter them because whenever new dataset load we clear all history?  
If so, we should write comments for this array describing why certain actions are excluded. (We can group actions why their reasons for exclusion.) 



2) Why is DATA_INLINE_RECEIVE here but DATASET_URL_RECEIVE not here?

3) Why should SET_CONFIG be excluded?  (Add comment)

4) (minor) line break here?  

```
const excludeActionsInHistory = [
   DATASET_INLINE_RECEIVE,
...
```